packages/twenty-website/src/app/releases/page.tsx

```diff
@@ -19,20 +25,37 @@ export const metadata: Metadata = {
 export const dynamic = 'force-dynamic';
 
 const Home = async () => {
-  const releases = await getReleases();
-  const mdxReleasesContent = await getMdxReleasesContent(releases);
+  const githubReleases = (await findAll(
+    githubReleasesModel,
+    desc(pgGithubReleasesModel.publishedAt),
+  )) as GithubReleases[];
+
+  const latestGithubRelease = githubReleases[0];
```

Comment:
If you just want to fetch the latest, you shouldn't fetch all results. Filtering should be done on the database side (performance issue, imagine if you had 1M release)